knative-operator/deploy/resources/knativekafka/testdata/config-logging.yaml

```diff
@@ -25,5 +25,5 @@ data:
         "callerEncoder": ""
       }
     }
-  loglevel.controller: "info"
+  loglevel.controller: "debug"
```

Comment:
Is this supposed to change?

Reply:
I think someone added that for better logs.

I can change here, or on a follow up PR

Reply:
fair enough to keep it as is too, I was just wondering if it has any effect to begin with, as it's in a testdata folder.

Reply:
will revert